### PR TITLE
UIE-200 Ajax cleanup pt6

### PR DESCRIPTION
**Pull request by msilva-broad**
- broke out Submissions sub area out of ajax.ts and into sub-module.
 - only minimal conversion to Typescript.

### What
-

### Why
-

### Testing strategy

- [ ]

### CODE REVIEW

File: src/libs/ajax/submissions/Submissions.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import _ from 'lodash/fp';
+import { authOpts } from 'src/auth/auth-session';
+import { fetchRawls } from 'src/libs/ajax/ajax-common';
+
+export const Submissions = (signal?: AbortSignal) => ({
```
Comment (msilva-broad) [inline src/libs/ajax/submissions/Submissions.ts]: I found no current uses of this.... Should I completely remove Submissions module?

Comment (aednichols) [reply src/libs/ajax/submissions/Submissions.ts]: Yes, see https://github.com/DataBiosphere/terra-ui/pull/1979

Comment (msilva-broad) [reply src/libs/ajax/submissions/Submissions.ts]: ok, will remove it

**Review by cahrens — APPROVED** [review]

**Review by aednichols — APPROVED** [review]
Thanks for the cleanup!

**Pull request merged**